plugins/dependencies/src/main/kotlin/dependencies/Kotlin.kt

```diff
@@ -7,7 +7,19 @@ import org.gradle.kotlin.dsl.IsNotADependency
 @Incubating
 object Kotlin {
 
+    /**
+     * Kotlin Standard Library
+     * https://kotlinlang.org/api/latest/jvm/stdlib/
+     */
     val stdlib = Stdlib
+
+    /**
+     * the kotlin.test library provides annotations to mark test functions
+     * and a set of utility functions for performing assertions in tests, independently of the test framework being used.
+     *
+     * https://kotlinlang.org/api/latest/kotlin.test/
+     * https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/
```

Comment:
These dangling links are not clickable in the IDE. Let's avoid that and always have text setting expectations on what the link is about.